### DISCUSSION TITLE
can_break_flag = 0 message with V1.3.3.8

### DISCUSSION THREAD

**Discussion by joergensen70** [Guilouz/Creality-Helper-Script-Wiki#529] [General]
Hi, after using the V1.3.3.8 I get messages like can_break_flag = 0/1/2...
Doesn't seem to have a negative impact but what is the root cause or is there a way to hide these messages?

By the way, really awesome work again, nice features and I really love the improvement with the virtual buttons and Kamp :-) Thank you so much for this great work.

**Comment by tent228** [comment]
What does can-break-flag really mean? Today, KAMP didn't have enough space for the lines, so he used a creative line. After drawing the line, it showed can-break-flag=3, and then paused directly. I manually resume it for a while...
